src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -182,20 +182,32 @@ public void register() {
 	public void register(boolean slimefun) {
 		addon = !slimefun;
 		try {
+			// Check if this item must be handled by Slimefun or return to its vanilla behavior
+			if(vanilla) SlimefunStartup.getItemCfg().setDefaultValue(this.name + ".ignore-slimefun-modifications", false);
+			if(SlimefunStartup.getItemCfg().getBoolean(this.name + ".ignore-slimefun-modifications")){
```

Comment:
The vanilla boolean acts like the "addon" boolean. If it is false, it will stay to false unless an addon does `item.setVanilla(true);`.
But I'll change it to what I said earlier. This will be more understandable